### PR TITLE
Issue 3128 + Issue 3112

### LINKED ISSUE #3112: BBB - install-update-from-epc.sh deletes /update on boot
not a show stopper, but the script and so the service should only be called from ePC and should not independently start on boot.

### LINKED ISSUE #3128: BBB: Update process messaging does not start at load time
Update process (either on the device or per WebUI Upload) does not emit any messages on the C15 until the update is copied, unpacked and has actually started.

Coping/Unpacking may take several 10s of seconds and during that period the C15 is non-responsive and appears to have crashed, causing confusion.... until it finally displays the "Starting Update" message.

The update message should therefore be started before the archive is copied, reading "Unpacking Update..." or similar

### PR DESCRIPTION
**Pull request by schlenkibus**
closes #3128
closes #3112

### CODE REVIEW

File: projects/bbb/scripts/install-update-from-epc.sh.in [hunk 1]
```diff
@@ -39,11 +39,10 @@ run_update() {
     freeze
 }
 
-
 main() {
+    @C15_INSTALL_PATH@/text2soled/text2soled multitext "Copying the Update...@SC1" "DO NOT SWITCH OFF!@SC2" "Copying the Update...@BC2" "DO NOT SWITCH OFF!@BC3"
     echo "install-update-from-epc.sh" >> /tmp/update.log
```
Comment (KlausStrohhaecker) [inline projects/bbb/scripts/install-update-from-epc.sh.in:44]: Suggest use string
"Unpacking C15 update..."

Comment (KlausStrohhaecker) [inline projects/bbb/scripts/install-update-from-epc.sh.in:44]: Suggest use string
"Unpacking C15 update..."

File: projects/bbb/scripts/install-update.sh.in [hunk 2]
```diff
@@ -34,9 +34,9 @@ run_update() {
     freeze
 }
 
-
 main() {
     if [ -e /mnt/usb-stick/nonlinear-c15-update.tar ]; then
+        @C15_INSTALL_PATH@/text2soled/text2soled multitext "Copying the Update...@SC1" "DO NOT SWITCH OFF!@SC2" "Copying the Update...EPC@BC2" "DO NOT SWITCH OFF!@BC3"
         unpack_update && check_preconditions && run_update
```
Comment (KlausStrohhaecker) [inline projects/bbb/scripts/install-update.sh.in:40]: Suggest use string
"Unpacking C15 update..."

Comment (KlausStrohhaecker) [inline projects/bbb/scripts/install-update.sh.in:40]: Suggest use string
"Unpacking C15 update..."

**Review by KlausStrohhaecker — APPROVED** [review]
Working OK here

**Review by KlausStrohhaecker — APPROVED** [review]

**Comment by KlausStrohhaecker** [comment]
I'd like to also merge this into `main` asap.
This is sort of an isolated feature and can be well tested in isolation and I see no reason to wait for `milestone-synthengine-update`

**Review by hhoegelo — APPROVED** [review]

**Pull request merged**